src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -477,6 +477,8 @@ private static string GetLocalPreferenceValue(string shellId, ExecutionPolicySco
 
 #endregion execution policy
 
+        private static bool _SaferIdentifyLevelApiSupported = true;
```

Comment:
From CodeFactor - Variable names and private field names must start with a lower-case letter.

Reply:
will change